src/index.js

```diff
@@ -15,3 +25,27 @@ module.exports = async function ({ github, context, inputs, packageVersion }) {
 
   logError('Unsupported event')
 }
+
+async function getBumpedVersionNumber({ inputs }) {
+  const newVersion =
+    inputs.semver === autoInput ? await getAutoBumpedVersion() : inputs.semver
+
+  const run = runSpawn()
+  await run('npm', ['version', '--no-git-tag-version', newVersion])
+  return await run('npm', ['pkg', 'get', 'version'])
+}
+
+async function getAutoBumpedVersion() {
+  try {
+    const { releaseType } = await conventionalRecommendedBumpAsync({ config })
```

Comment:
why don't we inline the require so we don't even require the package unless this function is executed?

Reply:
ok, this is nice because it's simple, but do we know what it's actually doing? I mean how do they know which commits to pick?

Reply:
@sameer-coder ^ this is the comment I was referring to. we need a way to specify the base tag/commit from which to generate the list of commits to consider. the package I believe currently doesn't support this, let's send them a PR